### PR TITLE
Update optionsmenu

### PR DESCRIPTION
**Pull request by IceflowRE**
Add
- guitooltip in advanced control menu looks now the current bind

Remove
- waterenv (not longer in use)
- lightmodels  (not longer in use)
- glowmodels  (not longer in use)

### CODE REVIEW

**Comment by bonifarz** [comment]
the tooltips are a good idea - are you sure you do not want to specify a width (arg2) for these?

**Comment by bonifarz** [comment]
Sorry for my concerns with the tooltips, variable width is fine here. But....

Is *waterenv* no longer in use? There's still *waterenvmap* in src/engine/water.cpp - is that what the checkbox @IceflowRE removed should be?
I don't know about light and glow options for models - is it correct to remove these?

**Comment by IceflowRE** [comment]
The "vars" i removed arent used by RE. (You can even save whatever you want in them) and i have take a look at waterenv, it looks like waterenv was never existing (doenst exist in 1.4.5 too) or only short in some dev versions. Waterenvmap is there yes, but i think we need no setting for this var because if its on or off, i see no difference and the fps is the same.

**Comment by bonifarz** [comment]
You or me cannot really tell, let us wait what someone with better engine knowledge will say.

Another thought: As you were addressing the advanced controls tab: I realize it can imply a significant drop in frame rate, as there are **many** buttons in this one. Maybe it is better to move it to a separate menu (newgui), so you don'tget this drop in fps when using the other tabs of the controls menu?

File: config/menus/options.cfg [hunk 1]
```diff
@@ -87,8 +87,6 @@ newgui options_graphics [
                 guicheckbox "^fyrefraction" waterrefract
                 guistrut 1
                 guicheckbox "^foreflection" waterreflect
-                guistrut 1
-                guicheckbox "^fgenvmap" waterenv
```
Comment (qreeves) [inline config/menus/options.cfg]: This variable is actually now called `waterenvmap`.

File: config/menus/options.cfg [hunk 2]
```diff
@@ -196,12 +194,8 @@ newgui options_graphics [
                 guiradio "^fyhigh quality" maxtexsize 0
             ]
             guilist [   // models
-                guicheckbox "^fglighting" lightmodels
-                guistrut 1
                 guicheckbox "^fgreflection" envmapmodels
                 guistrut 1
-                guicheckbox "^fgglow" glowmodels
-                guistrut 1
                 guicheckbox "^fybumpmap" bumpmodels
```
Comment (qreeves) [inline config/menus/options.cfg]: You could *maybe* added a slider here to control `fullbrightmodels` instead.

Comment (shirepirate) [inline config/menus/options.cfg]: >You could maybe add a slider here to control fullbrightmodels instead.
i added that to gameplay before, as there don't seem to be any other sliders in this GUI

**Comment by shirepirate** [comment]
>Maybe it is better to move it to a separate menu (newgui), so you don'tget this drop in fps when using the other tabs of the controls menu?

use guivisibletab for that

**Comment by IceflowRE** [comment]
The fullbrightmodels are controled in the gameplay menu already and here an guislider.. looks not very well i think.

**Comment by bonifarz** [comment]
poor fps on advanced controls tooltips 
> use guivisibletab for that

Mhm, no, I think it is better to clean up the code. My advanced controls menu was a moderate waste of resources, and the tooltips just make that even worse. I'll see if storing the binds in local vars when opening the menu helps. Should be much more efficient.

**Comment by shirepirate** [comment]
>I think it is better to clean up the code

definitely, but no reason to place it in a separate GUI as previously mentioned when there is guivisibletab

**Comment by bonifarz** [comment]
For a work-around, guivisibletab is okay, sure.
First attempt to store binds did not help. But we could try a slider layout to reduce the number of visible buttons. *shrug*

**Comment by bonifarz** [comment]
> - time race = race (update to an newer commit change)

I think this is where you may consider starting a new pull request? At least, that's what I do in such a case, although I'm not sure if merging this stuff would cause any trouble. It's just a single file to push anyway.

**Pull request merged**